LanguageExt.Core/Utility/Reflect.cs

```diff
@@ -30,7 +30,7 @@ public static IEnumerable<FieldInfo> GetPublicInstanceFields<A>(params Type[] ex
 #if !COREFX13
                                     .OrderBy(p => p.MetadataToken)
 #endif
-                                    .Where(p => p.CanRead && p.GetMethod.IsPublic)
+                                    .Where(p => p.CanRead && p.GetMethod.IsPublic && !p.GetAccessors(true)[0].IsStatic)
```

Comment:
Could `GetAccessors` ever return an empty array?  Would it be better to call: 

`!p.GetAccessors(true).HeadOrNone().Map(x => x.IsStatic).IfNone(true)`

Because then if `GetAccessors` ever did return an empty array then the property would be filtered out.

Reply:
It can return zero

![image](https://user-images.githubusercontent.com/17650/31925453-08c9148a-b889-11e7-9262-e28198c8a141.png)

https://msdn.microsoft.com/en-us/library/ff3b842x(v=vs.110).aspx

will fix.

Reply:
Actually.. When you pass **true** to GetAccessors then the array will always have two elements

https://msdn.microsoft.com/en-us/library/afadtey7(v=vs.110).aspx

> An array whose elements reflect the get and set accessors of the property reflected by the current instance. **If nonPublic is true, this array contains public and non-public get and set accessors.** If nonPublic is false, this array contains only public get and set accessors. If no accessors with the specified visibility are found, this method returns an array with zero (0) elements.

Still just to be safe I'll fix the code